subprojects/docs/src/samples/build-organization/standalone-precompiled-plugins/kotlin/src/main/kotlin/com.example.service.gradle.kts

```
@@ -0,0 +1,30 @@
+// tag::plugins[]
+plugins {
+    id("com.example.java-convention")
+}
+// end::plugins[]
+
+val integrationTest by the<SourceSetContainer>().creating {
+    compileClasspath += sourceSets["main"].output + sourceSets["test"].output
+    runtimeClasspath += sourceSets["main"].output + sourceSets["test"].output
```

:x: type safe accessors to source sets should be usable here. It needs an extra `.get()` because the accessors are providers.
```suggestion
    compileClasspath += sourceSets.main.get().output + sourceSets.test.get().output
    runtimeClasspath += sourceSets.main.get().output + sourceSets.test.get().output
```